temporalcli/commandsmd/commands.md

```diff
@@ -560,533 +1300,836 @@ Includes options set for [overlap-policy](#options-set-for-overlap-policy).
+### temporal task-queue get-build-id-reachability: Build ID availability
 
-This command can tell you whether or not Build IDs may be used for new, existing, or closed workflows. Both the '--build-id' and '--task-queue' flags may be specified multiple times. If you do not provide a task queue, reachability for the provided Build IDs will be checked against all task queues.
+Shows if a given Build ID can be used for new, existing, or closed workflows
+in Namespaces that support Worker versioning:
+
+```
+temporal task-queue get-build-id-reachability --build-id "2.0"
+```
+You can specify the '--build-id' and '--task-queue' flags multiple times.
+When '--task-queue' is omitted, checks Build ID reachability against all
+task queues.
 
 #### Options
 
-* `--build-id` (string[]) - Which Build ID to get reachability information for. May be specified multiple times.
-* `--reachability-type` (string-enum) - Specify how you'd like to filter the reachability of Build IDs. Valid choices are `open` (reachable by one or more open workflows), `closed` (reachable by one or more closed workflows), or `existing` (reachable by either). If a Build ID is reachable by new workflows, that is always reported. Options: open, closed, existing. Default: existing.
-* `--task-queue`, `-t` (string[]) - Which Task Queue(s) to constrain the reachability search to. May be specified multiple times.
+* `--build-id` (string[]) -
+  One or more Build ID strings.
+  Can be passed multiple times.
+* `--reachability-type` (string-enum) -
+  Reachability filter.
+  `open`: reachable by one or more open workflows.
+  `closed`: reachable by one or more closed workflows.
+  `existing`: reachable by either.
+  New Workflow Executions reachable by a Build ID are always reported.
+  Options: open, closed, existing.
+  Default: existing.
+* `--task-queue`, `-t` (string[]) -
+  List of Task Queues.
+  Constrains the reachability search.
+  Can be passed multiple times.
+
+### temporal task-queue get-build-ids: Build ID versions
+
+Fetches sets of compatible Build IDs for specified Task Queues and displays
+their information:
 
-### temporal task-queue get-build-ids: Fetch the sets of worker Build ID versions on the Task Queue.
+```
+temporal task-queue get-build-ids \
+    --task-queue "YourTaskQueue"
+```
 
-Fetch the sets of compatible build IDs associated with a Task Queue and associated information.
+Limited to Namespaces that enable Worker versioning.
 
 #### Options
 
-* `--task-queue`, `-t` (string) - Task queue name. Required.
-* `--max-sets` (int) - Limits how many compatible sets will be returned. Specify 1 to only return the current default major version set. 0 returns all sets. (default: 0). Default: 0.
+* `--task-queue`, `-t` (string) -
+  Task queue name.
+  Required.
+* `--max-sets` (int) -
+  Max return count.
+  Use 1 for default [major-version](https://en.wikipedia.org/wiki/Software_versioning) set.
```

Comment:
Sadly, Markdown links don't work / aren't stripped properly in flags.

Reply:
Oof, removing. Making note about that.